localgov_base.theme

```diff
@@ -69,7 +70,16 @@ function localgov_base_preprocess_page(&$variables) {
       continue;
     }
     $copy = $variables['page'][$region];
-    $rendered = \Drupal::service('renderer')->renderPlain($copy);
+
+    /** @var \Drupal\Core\Render\RendererInterface $renderer */
+    $renderer = \Drupal::service('renderer');
+    $rendered = DeprecationHelper::backwardsCompatibleCall(
+      currentVersion: \Drupal::VERSION,
+      deprecatedVersion: '10.3',
+      currentCallable: fn() => $renderer->renderInIsolation($copy),
+      deprecatedCallable: fn() => $renderer->renderPlain($copy),
+    );
+
```

Comment:
I'm guessing this is something we need to fix for D11 rather than just marking it as deprecated?

If so, should we try fix it now, or create a follow up for it?

Reply:
Correct, this allows support for both D10 and D11. renderInIsolation was added in Drupal 10.3 so the depreaction helper allows us to support <10.3 and >=10.3 without requiring multiple versions of the module. 

When the deprecation helper is used it does not flag the deprecation as it is wrapped in the helper, however when we decide to no longer support <10.3 (Drupal 10 is supported until mid/late 2026) we can the remove the helper completely.

Reply:
Great. Let's get this merged so and that follow up created. Thanks Lee.

===
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.